files/docker-container-logrotate

```diff
@@ -0,0 +1,9 @@
+/var/lib/docker/containers/*/*.log {
+  rotate 0
+  hourly
+  notifempty
+  nocompress
+  size 20k
```

Comment:
I am not suggesting to do this hourly. But I wanted to understand if we knew what this setting is. Anybody else who looks at this will get confused. You are free to push this if you think it is fine.